### PR TITLE
Add player coordinates to debug stats (H key)

### PR DESCRIPTION
**Pull request by Loryhoof**
image

### CODE REVIEW

**Comment by amnaarshed4224** [comment]
works fine

.

File: stats.js [hunk 1]
```diff
@@ -41,15 +44,22 @@ export var Stats = function () {
...
 			frames++;
-		
-	        if (Date.now() > lastTime + 1000) {
 
+			var i = 0;
 
-				var i	= 0;
-				msTexts[i++].textContent = "FPS: "	+ Math.round((frames * 1000) / (Date.now() - lastTime));
+			// Update every frame
+			if(localPlayer) {
```
Comment (avaer) [inline stats.js:51]: Is this update called even when the stats panel is closed? This has allocations and isn't free so I don't want to take away CPU from the app by default.

**Review by avaer — COMMENTED** [review]
One question, but it looks good + good idea.

**Comment by Loryhoof** [comment]
d
Only updating when panel is open

**Comment by Loryhoof** [comment]
https://github.com/webaverse/app/blob/b5d9a8f569e2b51341268e0cfe02869d3b772dc7/webaverse.js#L294

**Review by avaer — APPROVED** [review]

**Pull request merged**